api/PasswordCredential.json

```diff
@@ -0,0 +1,158 @@
+{
+  "api": {
+    "__compat": {
+      "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential",
+      "support": {
+        "chrome": {
+          "version_added": "51"
+        },
+        "webview_android": {
+          "version_added": "51"
+        },
+        "chrome_android": {
+          "version_added": "51"
+        }
+      },
+      "status": {
+        "experimental": true,
+        "standard_track": true,
+        "deprecated": false
+      }
+    },
+    "additionalData": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/PasswordCredential/additionalData",
+        "support": {
+          "chrome": {
+            "version_added": "51",
+            "version_removed": "60"
```

Comment:
This was actually removed in 64, as were `idName` and `passwordName`. Please change all for all 3 flavors of Chrome.

https://chromiumdash.appspot.com/commit/57b2dc23b962feeadd2dcf60a399c9f38d2087e3